Add information about SVG transform support on <svg> root (#39964)
mdn/content: files/en-us/web/svg/reference/attribute/transform/index.md

> [!NOTE]

> > As a presentation attribute, `transform` also has a CSS property counterpart: {{cssxref("transform")}}. When both are specified, the CSS property takes priority. However, be aware that there are some differences in syntax between the CSS property and the attribute. See the documentation for the CSS property {{cssxref('transform')}} for the specific syntax to use in that case.

Reviewer: I think this is a little redundant.

```suggestion
> As a presentation attribute, `transform` also has a CSS property counterpart: {{cssxref("transform")}}. When both are specified, the CSS property takes priority. Note that there are some differences in syntax between the CSS property and the attribute!
```

## Elements

> In SVG 2, you can use the `transform` attribute on any SVG element inside the {{SVGElement('svg')}} root. In addition, some browsers support using `transform` on the `<svg>` root itself, which is convenient for applying transforms to an entire SVG image without the need for extra wrapper elements or CSS workarounds.

Reviewer: I'm worried this won't get updated when supported by all browser. Which is annoying, because I like the way you have worded this. 

We document the spec, so perhaps you might consider this. 

```suggestion
In SVG 2, you can use the `transform` attribute on any element, including the {{SVGElement('svg')}} root.
Note that using `transform` on the `<svg>` root is a newer feature, and you should check [browser compatibility](#browser_compatibility) for support.
Using `transform` on the `<svg>` root is convenient for applying transforms to an entire SVG image without the need for extra wrapper elements or CSS workarounds.

```
